pkg/microservice/aslan/server/rest/router.go

```diff
@@ -95,6 +95,7 @@ func (s *engine) injectRouterGroup(router *gin.RouterGroup) {
 	}
 
 	router.GET("/api/apidocs/*any", ginswagger.WrapHandler(swaggerfiles.Handler))
+	router.GET("/healthz", commonhandler.HandleCallback)
```

Comment:
Is this handler wrong?
The healthz API is closely related to the business, and there is usually no general handler.